addon/components/paper-toast.js

```diff
@@ -0,0 +1,101 @@
+import Ember from 'ember';
+import TransitionMixin from 'ember-css-transitions/mixins/transition-mixin';
+import PaperToastBounds from './paper-toast-bounds';
+const {
+  Component,
+  computed,
+  run
+} = Ember;
+const {
+  later,
```

Comment:
We decided to not destructure past the first level. See contributing.md. (If it isn't up-to-date, let me know). So you would code `run.later(`, not `later`.